.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE_FEATURE.md

```diff
@@ -1,10 +1,9 @@
 ---
-name: 
+name:
```

Comment:
why github templates are in this PR ? :D

Reply:
These are added due to the merging the development branch into this branch. You can refer to [this commit](https://github.com/LiskHQ/lisk-sdk/pull/4046/commits/b2d5f5a0e925ddf52d707a6ac53eeead4d1ae0e0) to see. Do you know how to fix this? Linking: @shuse2